go.mod

```diff
@@ -28,7 +28,6 @@ require (
 	github.com/google/go-github/v50 v50.2.0
 	github.com/google/uuid v1.5.0
 	github.com/hinshun/vt10x v0.0.0-20220301184237-5011da428d02
-	github.com/jba/templatecheck v0.7.0
```

Comment:
No longer used?

Reply:
replaced the templatecheck.Check (which only checks for fields match) with templatewriter.Print (which does the fields match + prints to a buffer)

Reply:
Awesome change!